docker/benchmark.yml

```diff
@@ -1,4 +1,3 @@
-version: "3.4"
```

Comment:
Removed the version, as this seems to be the most recent convention for `docker compose`: https://docs.docker.com/compose/compose-file/compose-versioning/